skimage/color/colorconv.py

```diff
@@ -56,7 +56,9 @@
 from warnings import warn
 import numpy as np
 from scipy import linalg
-from ..util import dtype, dtype_limits
+from ..util import dtype, dtype_limits, img_as_ubyte
+from matplotlib import pyplot as plt
```

Comment:
MPL is a required dependency: https://github.com/scikit-image/scikit-image/blob/master/requirements.txt#L1